### DISCUSSION TITLE
Cannot use `openChange` of `TuiMultiSelectComponent`

### DISCUSSION THREAD

**Discussion by erzenh** [taiga-family/taiga-ui#6792] [Q&A] [answered]
I want to use `openChange` of `TuiMultiSelectComponent` to check when dropdown is opened, but `hostedDropdown` of `TuiMultiSelectComponent` is private.

https://github.com/taiga-family/taiga-ui/blob/b16dbc22a430c3ebfb267390b0bd264aa9cded80/projects/kit/components/multi-select/multi-select.component.ts#L84

**Comment by waterplea** [comment, marked as answer]
You can use `[(tuiDropdownOpen)]` from `TuiDropdownModule`:
![image](https://github.com/taiga-family/taiga-ui/assets/11832552/9e406e6d-0a92-44bb-80e9-f370b0e091b6)
